Gemfile

```diff
@@ -1,2 +1,5 @@
 source "https://rubygems.org"
 gemspec
+
+gem "rubocop-rspec", "~> 1.15.1"
+gem "rails-controller-testing"
```

Comment:
Please do not edit the `Gemfile`. Please edit `Appraisals` instead. But, I don't think any changes to gems are necessary.